spk/rutorrent/src/conf/resource

```diff
@@ -0,0 +1,8 @@
+{
+    "php56": {
```

Comment:
Officially rutorrent seems to be PHP 5.6 only... Didn't bother adding other PHP versions in case it wouldn't work. I'll leave that to some other user who would be interested in testing it ;)